models/Host.v1.yaml

```
@@ -1,5 +1,5 @@
 title: Host
-description: A client host assigned to a Signin
+description: "The root of the Host type's schema."
```

Should revert this file's changes
```suggestion
description: A client host assigned to a Signin
```

---

Is `A client host assigned to a Signin` an appropriate description for a Host?